application/controllers/HostsController.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/* Icinga Graphite Web | (c) 2022 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Graphite\Controllers;
+
+use GuzzleHttp\Psr7\ServerRequest;
+use Icinga\Module\Graphite\Web\Controller\IcingadbGraphiteController;
+use Icinga\Module\Graphite\Web\Controller\TimeRangePickerTrait;
+use Icinga\Module\Graphite\Web\Widget\IcingadbGraphs;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Icingadb\Web\Control\SearchBar\ObjectSuggestions;
+use Icinga\Web\Url;
+use ipl\Html\HtmlString;
+use ipl\Stdlib\Filter;
+use ipl\Web\Control\LimitControl;
+use ipl\Web\Control\SortControl;
+
+class HostsController extends IcingadbGraphiteController
+{
+    use TimeRangePickerTrait;
+
+    public function indexAction()
+    {
+        if (! $this->useIcingadbAsBackend) {
+            $params = urldecode($this->params->get('legacyParams'));
+            $this->redirectNow(Url::fromPath('graphite/list/hosts')->setQueryString($params));
+        }
+
+        // shift graph params to avoid exception
+        $graphRange = $this->params->shift('graph_range');
+        $baseFilter = $graphRange ? Filter::equal('graph_range', $graphRange) : null;
+        foreach ($this->graphParams as $param) {
+            $this->params->shift($param);
+        }
+
+        $this->setTitle(t('Hosts'));
```

Comment:
I forgot this. `setTitle` has changed. You'll need to call `addTitleTab` now. I've rebased https://github.com/Icinga/ipl-web/pull/69, otherwise `addTitleTab` is missing.

Reply:
The call to `setTitle` in method `searchEditorAction` does *not* need to change.